app/models/file_depot.rb

```diff
@@ -12,18 +12,13 @@ class FileDepot < ApplicationRecord
   attr_accessor         :file
 
   def self.supported_depots
-    @supported_depots ||= descendants.each_with_object({}) { |klass, hash| hash[klass.name] = Dictionary.gettext(klass.name, :type => :model, :notfound => :titleize, :translate => false) }.freeze
+    @supported_depots ||= descendants.each_with_object({}) { |klass, hash| hash[klass.name] = klass.display_name }.freeze
```

Comment:
@NickLaMuro However, to your point, this code is caching a display_name as well.  @mzazrivec, will that cause problems?

Reply:
@Fryguy you're right about the caching problem here. Fixed.